src/tcp_wrap.cc

```diff
@@ -428,7 +428,7 @@ void TCPWrap::Connect(const FunctionCallbackInfo<Value>& args) {
 
   assert(args[0]->IsObject());
   assert(args[1]->IsString());
-  assert(args[2]->Uint32Value());
+  assert(args[2]->Uint32Value() >= 0);
```

Comment:
Ah, you're right. I added the `>=` because when `Uint32Value()` was equal to 0, the assertion failed.